### ISSUE TITLE
Steam version does not show cloud saves and local saves will not load.

### ISSUE THREAD

**Issue by GGustos** [MelvorIdle/melvoridle.github.io#988]
I am using the Steam version of Melvor Idle and have 570 hour playtime so far.
This morning when I loaded the game, there were no Cloud saves.  I tried loading a local save and received a popup that says the following:
"Umm.... Looks like the game broke
:(
Oh dear... it looks like the game failed to load.
Maybe you should let the dev know about this error:
Fix my Save"

I tried pressing the "Fix my Save" button and it did something with the file and took me back to the game load screen.  When I tried loading the game, I again received the popup with the same error about the game being broke.

I tried uninstalling the Steam game and removed all of the files.  When I opened the game, I still encountered no cloud saves and the local saves are not working.

I have four different characters.  I am able to import each of them using the website.  None of them are working on Steam.

Attached are copies of the local saves and screenshots of the error.  Please let me know if you need anything else.

Thank you

![20210608072434_1](https://user-images.githubusercontent.com/85573827/121219430-87fa8c80-c838-11eb-8a94-a0d7255c2011.jpg)
![20210608072412_1](https://user-images.githubusercontent.com/85573827/121219434-88932300-c838-11eb-9cad-d71ab5bcbd66.jpg)

[melvoridlesave_G.txt](https://github.com/MelvorIdle/melvoridle.github.io/files/6618031/melvoridlesave_G.txt)
[melvoridlesave_AdventureGus.txt](https://github.com/MelvorIdle/melvoridle.github.io/files/6618033/melvoridlesave_AdventureGus.txt)
[melvoridlesave_ChaosGus.txt](https://github.com/MelvorIdle/melvoridle.github.io/files/6618034/melvoridlesave_ChaosGus.txt)
[melvoridlesave_Hard G.txt](https://github.com/MelvorIdle/melvoridle.github.io/files/6618035/melvoridlesave_Hard.G.txt)

**Comment by GGustos** [comment]
I just tried to create a new character.  I added the name and clicked "Begin your Journey" and it never loaded.
Now, all the game does now is shows the M with a red line rotating around it and doesn't load to the character select screen.

**Comment by Fuvias** [comment]
Have you tried reinstalling your client, maybe something went wrong in installation!

Alternatively, if the dev isn't able to get to you or may take awhile, you may have better chance on getting someone to help you more on the discord! ^-^

**Comment by GGustos** [comment]
I re-installed 2 times.  After the first uninstall/re-install, I was getting the same resulting error.
With the second uninstall/re-install, I also removed everything in the user save directory.  Deleting all the files in the user directory seemed to have cured whatever the problem was.
I have no idea what happened to start encountering the problem.  It was working fine when I went to bed and when I started up the game the next morning, there were no cloud saves and the local saves did not work.
All of the cloud saves are now available and the local saves are also working.
Thank you.
